src/index.html

```diff
@@ -1,12 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day calendar__day--date--1 calendar__day--start-mon"></div>
+      <div class="calendar__day calendar__day--date--2"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
```

redundant class
remove all these